examples/chart/teleport-cluster/values.yaml

```diff
@@ -54,6 +54,20 @@ acmeURI: ""
 # kubectl -n ${TELEPORT_NAMESPACE?} create secret generic license --from-file=/path/to/downloaded/license.pem
 enterprise: false
 
+# Configuration of the optional Teleport operator
+operator:
```

Comment:
I prefer the separation here
However, I think we are missing the `image` here (it currently exists as `operatorImage`)